packages/intrepid2/src/Orientation/Intrepid2_OrientationToolsDefCoeffMatrix_HCURL.hpp

```diff
@@ -210,48 +203,47 @@ getCoeffMatrix_HCURL(OutputViewType &output,
   // on the subcell spanning the same space as the bases \phi_j
 
   // Tangents t_j
-  ScalarViewType subcellTangents("subcellTangents", numSubcellBasis, subcellDim);
+  Kokkos::DynRankView<value_type,host_device_type> subcellTangents("subcellTangents", numSubcellBasis, subcellDim);
```

Comment:
We should be able to make Orientation run on device without much effort. What issues did you encounter?

Reply:
This is initialization. There is no gain from exporting this to device.

Reply:
What problem do you encounter without UVM, if you try to run this on device?